### PR TITLE
[15.0][ADD] purchase_receipt_expectation_manual

### PR DESCRIPTION
**Pull request by SilvioC2C**
Includes #1348

### CODE REVIEW

File: purchase_receipt_expectation_manual/wizards/purchase_order_manual_receipt.py [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+        self.ensure_one()
+        # Setting `checks_result` to "success" because the user is allowed to
+        # ignore warnings; then, we'll run pre-confirm checks and, if any of
+        # them fails, field `checks_result` should be changed
+        self.checks_result = "success"
+        errs = self._execute_pre_confirm_checks()
+        self.write(self._prepare_post_check_vals(errs))
+        if self.checks_result != "success":
+            # Some pre-confirm check failed: reopen the wizard
+            self.checks_result_msg += _("\n\nThese checks cannot be skipped!")
```
Comment (ivantodorovich) [inline purchase_receipt_expectation_manual/wizards/purchase_order_manual_receipt.py:85]:
```suggestion
            self.checks_result_msg += "\n\n" + _("These checks cannot be skipped!")
```
It's better for translations

**Review by ivantodorovich — APPROVED** [review]

**Review by ajaniszewska-dev — APPROVED** [review]

**Comment by dreispt** [comment]
Blocked by #1348

**Comment by CasVissers-360ERP** [comment]
@SilvioC2C 
Why not extending purchase_manual_delivery here?

**Pull request merged**